routing/src/prefix.rs

```diff
@@ -166,7 +167,7 @@ impl Prefix {
         T: TryInto<Prefix>,
         T::Error: Debug,
     {
-        val.try_into().expect("Invalid prefix")
+        val.try_into().unwrap()
```

Comment:
why?

Reply:
oh, I was debugging the test and the expect was eating the error message

Reply:
Oh, I never realised it would :exploding_head: